skrub/_gap_encoder.py

```diff
@@ -456,7 +479,14 @@ def fit(self, X, y=None):
         self._input_name = sbd.name(X)
         self._random_state = check_random_state(self.random_state)
         is_null = sbd.to_numpy(sbd.is_null(X))
-        X = sbd.to_numpy(X)
+        result = self._fit_transform(sbd.to_numpy(X), is_null)
+
+        self.scaler_ = total_standard_deviation_scaler(result)
```

Comment:
To me, the name "self.scaler_" makes me think that it is a scaling object, ie a transformer, and not a float.

I would prefer this value to be called "self.scale_" or maybe "self.scaling_factor_".